deploy/charts/kiali-operator/templates/clusterrole.yaml

```diff
@@ -0,0 +1,311 @@
+apiVersion: rbac.authorization.k8s.io/v1
```

Comment:
Here's what I suggest for this file. Because its essentially identical to our other clusterrole.yaml, you should indent it the same way; e.g. put the list items on the same level as the parent. 

That way we can do a diff and determine what, if anything, changed between one versus the other (this lets us more easily keep them in sync).

For example,

```
resources:
- configmaps
```

as opposed to:

```
resources:
  - configmaps
```

This way one can do a "diff" on this file and the other clusterrole.yaml and at least get a better view of what changed (otherwise, every line will look changed with the different indentation).